beta-pipelines/multiome/atac.wdl

```diff
@@ -299,31 +310,49 @@ task CreateFragmentFile {
     python3 <<CODE
 
 
+    atac_gtf = "~{atac_gtf}"
     barcodes_in_read_name = "~{barcodes_in_read_name}"
     bam = "~{bam}"
     bam_base_name = "~{bam_base_name}"
+    chrom_sizes = "~{chrom_sizes}"
+    
+    # Calculate chrom size dictionary based on text file
+    chrom_size_dict={}
+    with open('~{chrom_sizes}', 'r') as f:
+        for line in f:
+            key, value = line.strip().split()
+            chrom_size_dict[str(key)] = int(value)
 
     # if barcodes are in the read name, then use barcode_regex to extract them. otherwise, use barcode_tag
 
     if barcodes_in_read_name=="true":
       import snapatac2.preprocessing as pp
+      import snapatac2 as snap
       pp.make_fragment_file("~{bam}", "~{bam_base_name}.fragments.tsv", is_paired=True, barcode_regex="([^:]*)")
+      pp.import_data("~{bam_base_name}.fragments.tsv", file="~{bam_base_name}.metrics.h5ad", chrom_size=chrom_size_dict, gene_anno="~{atac_gtf}")
     elif barcodes_in_read_name=="false":
       import snapatac2.preprocessing as pp
+      import snapatac2 as snap
       pp.make_fragment_file("~{bam}", "~{bam_base_name}.fragments.tsv", is_paired=True, barcode_tag="CB")
+      pp.import_data("~{bam_base_name}.fragments.tsv", file="~{bam_base_name}.metrics.h5ad", chrom_size=chrom_size_dict, gene_anno="~{atac_gtf}")
 
     CODE
+    
+    echo printing directory
+    pwd
+    echo printing files
```

Comment:
Lines 340 to 344 can be removed. Or if we want to keep it -- maybe we can describe what were printing for better user experience.

Reply:
Removed them.